### DISCUSSION TITLE
4 Players Split Screen

### DISCUSSION THREAD

**Discussion by tautologias345** [enable3d/enable3d#296] [General]
How can I do this in Phaser? I don't want NodeJS.

**Comment by yandeu** [comment]
For Phaser questions, please use https://phaser.discourse.group/

**Comment by tautologias345** [comment]
I already know how in Phaser can I create 4 cameras adjusting the viewport.

https://phaser.discourse.group/t/how-can-i-do-4-players-split-screen/11877

I already know how to use gamepad input.

But about 3D camera, I just know how to have 1 camera. How to have 4 3D cameras?

**Reply by yandeu** [reply]
I have never used multiple cameras in three.js. Please have a look at: https://discourse.threejs.org/

**Comment by yandeu** [comment]
See https://enable3d.io/examples/first-person-shooter.html
